src/Routes.jsx

```diff
@@ -1,21 +1,88 @@
 import React, { Component } from 'react';
-import {
-  graphql,
-  createFragmentContainer,
-} from 'react-relay';
+import DeckGL, { GeoJsonLayer, IconLayer } from 'deck.gl';
 import propTypes from 'prop-types';
+import { createFragmentContainer, graphql } from 'react-relay';
+import muniRoutesGeoJson from './res/muniRoutes.geo.json';
+
+const atlasIcon = require('./res/icon-atlas.png');
+
+const routesLayer = new GeoJsonLayer({
+  id: 'muni-routes-geojson',
+  data: muniRoutesGeoJson,
+  filled: true,
+  stroked: false,
+  extruded: true,
+});
+
 
 class Routes extends Component {
+  getMarkers() {
```

Comment:
could we rename this to something like `getVehicleMarkers`? let's do stops in a future CR.